### ISSUE TITLE
Error when starting in Windows Terminal

### ISSUE THREAD

**Issue by makue** [PowerShell/PSReadLine#2999]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
When I start Powershell Core in Windows Terminal, I get an error message.
Error also appears when you are in the Powershell Core application.

The console cannot currently be used!

The error only came after the update to version 7.2.0.

### Exception

System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

### Screenshot

![image](https://user-images.githubusercontent.com/32456044/141094303-dd1c5e66-58bf-4d5b-a29a-f3fa6ddaab63.png)

### Environment data

```console
Version 7.2.0


No further commands are possible.
Console cannot be used.
```

### Steps to reproduce

Just start the program

### Expected behavior

I can use the shell normally.

### Actual behavior

After starting the console, an error message is displayed. You can't use the Core Console.

**Comment by makue** [comment]
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30

**Comment by iSazonov** [comment]
Please update to latest Beta4 version.

**Comment by makue** [comment]
I couldn't do the update because my console didn't accept a command.

I was able to solve the problem myself.
I deleted the PSReadLine model. Then I reinstalled Powerhsell Core.
Then I reinstalled the model.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
